README.md

```diff
@@ -1,2 +1,156 @@
-# snap-scan
+<a name="readme-top"></a>
+<h1 align='center'> SnapScan </h1>
+
+##Preview
+![Screenshot 2023-09-14 163634](https://github.com/IsmailChangezi/snap-scan/assets/96125730/dd0a4a23-b9a9-4473-b762-8cbabd0ff327)
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [🥙 Food On Rails ](#-Food-On_Rails-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+
+# SnapScan <a name="about-project"></a>
+
 SnapScan is a mobile application specially crafted to let the user have a count of their expanses. It provides the user with security and an impressive UI/UX. This app is built with Ruby On Rails.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Use database migrations**
+- **Tranction Scaning**
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/IsmailChangezi/snap-scan.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rails rspec spec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Video Explanation
+This is the link to the video explaination
+
+https://www.loom.com/share/My-Project-Journey-0daf09dd9d7242559ce9247b87e25aa8?sid=058c64bf-1f3c-4da9-85a3-62b53dcca18e
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [ ] Kindly deploy the application using Render, Netlify or GitHub Pages, you should have a link to the deployed project  in the README.